src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetJudicialTaskControllerCFTTest.java

```diff
@@ -0,0 +1,117 @@
+import static uk.gov.hmcts.reform.wataskmanagementapi.controllers.request.enums.TaskAttributeDefinition.TASK_ROLE_CATEGORY;
+import static uk.gov.hmcts.reform.wataskmanagementapi.controllers.request.enums.TaskAttributeDefinition.TASK_TITLE;
+import static uk.gov.hmcts.reform.wataskmanagementapi.controllers.request.enums.TaskAttributeDefinition.TASK_TYPE;
+
+public class GetJudicialTaskControllerCFTTest extends SpringBootFunctionalBaseTest {
+    private static final String TASK_INITIATION_ENDPOINT_BEING_TESTED = "task/{task-id}";
+    private static final String ENDPOINT_BEING_TESTED = "task/{task-id}";
+    private Headers authenticationHeaders;
+
+    @Before
+    public void setUp() {
+        authenticationHeaders = authorizationHeadersProvider.getJudgeAuthorization("wa-ft-test-r2-");
+
+    }
+
+    @Test
+    public void should_return_a_200_with_allocateHearingJudge_task_and_standard_grant_type() {
+
+        TestVariables taskVariables = common.setupTaskAndRetrieveIds("allocateHearingJudge");
+        String taskId = taskVariables.getTaskId();
+
+        initiateTaskForJudicial(taskVariables);
+
+        common.setupCFTJudicialOrganisationalRoleAssignment(authenticationHeaders,
+                                                            GrantType.STANDARD.name(), taskVariables.getCaseId());
+
+        Response result = restApiActions.get(
+            ENDPOINT_BEING_TESTED,
+            taskId,
+            authenticationHeaders
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .and().contentType(MediaType.APPLICATION_JSON_VALUE)
+            .body("task.id", notNullValue())
+            .body("task.name", notNullValue())
+            .body("task.type", notNullValue())
+            .body("task.task_state", notNullValue())
+            .body("task.task_system", notNullValue())
+            .body("task.security_classification", notNullValue())
+            .body("task.task_title", notNullValue())
+            .body("task.created_date", notNullValue())
+            .body("task.due_date", notNullValue())
+            .body("task.location_name", notNullValue())
+            .body("task.location", notNullValue())
+            .body("task.execution_type", notNullValue())
+            .body("task.jurisdiction", notNullValue())
+            .body("task.region", notNullValue())
+            .body("task.case_type_id", notNullValue())
+            .body("task.case_id", notNullValue())
+            .body("task.case_category", equalTo("Protection"))
+            .body("task.case_name", notNullValue())
+            .body("task.auto_assigned", notNullValue())
+            .body("task.warnings", notNullValue())
+            .body("task.permissions.values", hasItems("Read", "Refer", "Execute"))
+            .body("task.permissions.values", hasSize(3))
+            .body("task.description", notNullValue())
+            .body("task.role_category", equalTo("ADMINISTRATOR"));
```

Comment:
ADMINISTRATOR?

This might need rebase and refactoring also after 1104 is merged. (ADMINISTRATOR is now ADMIN)